widgets/data_editor.py

```diff
@@ -827,12 +827,17 @@ def setup_widgets(self):
         self.end = self.add_multiple_decimal_input("End", "end", ["x", "y", "z"],
                                                      -inf, +inf)
 
-        self.unk1 = self.add_integer_input("Unknown", "unk1",
+        self.unk1 = self.add_integer_input("Shortcut Point ID", "unk1",
                                            MIN_UNSIGNED_BYTE, MAX_UNSIGNED_BYTE)
-        self.unk2 = self.add_checkbox("Unknown Flag", "unk2",
-                                      0, 1)
-        self.unk3 = self.add_checkbox("Unknown Flag 2", "unk3",
+        self.unk3 = self.add_checkbox("Double-sided", "unk3",
                                            0, 1)
+        self.unk4 = self.add_checkbox("Lap Checkpoint🧩", "unk4",
+                                           0, 1)
+        self.unk4.toggled.connect(self.catch_text_update)
+
+        self.unk1.setToolTip(ttl.checkpoints["Shortcut Point ID"])
+        self.unk3.setToolTip(ttl.checkpoints["Double-sided"])
+        self.unk4.setToolTip(ttl.checkpoints["Lap Checkpoint🧩"])
```

Comment:
I'd probably omit the emoji in the tool tip key name; it's not something that is exposed to the user.

As long as we have it in the label, and in the actual tool tip text, that's enough (I think).